src/pages/termsOfUse.tsx

```diff
@@ -16,9 +16,318 @@ export const TermsOfUsePage: FunctionComponent = () => {
       </Helmet>
       <Page title="Terms Of Use">
         <div className="bg-white drop-shadow-lg rounded-xl h-full w-full px-6 py-4 mt-4">
-          <ReactMarkdown components={{ p: ({ ...props }) => <p className={"mb-4 break-word"} {...props} /> }}>
-            {termsOfUse.body}
-          </ReactMarkdown>
+          <h4>1. General</h4>
```

Comment:
rather than manually managing numbering, use [CSS counters](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Counter_Styles/Using_CSS_counters#Nesting_counters)? this way we can retain the markdown

Reply:
https://github.com/TradeTrust/tradetrust-website/pull/650